com.note_slayer.NoteSlayer.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.note_slayer.NoteSlayer
+base-version: "23.08"
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: noteslayer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+separate-locales: false
+rename-desktop-file: noteslayer.desktop
+rename-icon: noteslayer
+modules:
+  - name: noteslayer
+    buildsystem: simple
+
+    sources:
+      - type: file
+        url: https://github.com/X-SLAYER/note_slayer_public/releases/download/V0.1/note_slayer_linux.deb
+        sha256: 5aee6f197601b82b65355dad919477789b466a27ae7c8bbd33d19a90d0d01270
+        only-arches: ["aarch64"]
```

Comment:
so how is this gonna work on x86_64?

Since from that change you are telling us it was aarch64 all along.

Reply:
yes